### PR TITLE
Update setup wizard and first time setup

### PR DESCRIPTION
**Pull request by bhalsey**
Update screenshots and text to match updated Setup Wizard and new First Time Setup.

Also consolidated blog screenshots.  (did not update the fly io one b/c it has a different location bar url from the rest of them).

### CODE REVIEW

File: astro/src/content/blog/single-sign-on-django-fusionauth.mdx [hunk 1]
```diff
@@ -114,7 +114,7 @@ Password: (the default value is fine)
 
 Once the setup is complete, you'll be automatically redirected to the "FusionAuth Setup Wizard", as seen here:
```
Comment (andrewpai) [inline astro/src/content/blog/single-sign-on-django-fusionauth.mdx:115]: Not part of this PR, but in other places we just use Setup Wizard in caps with no quotes, should probably do that here, too.

File: astro/src/content/docs/get-started/download-and-install/setup-wizard.mdx [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Setup Wizard & First Login
+title: Setup Wizard & First Time Setup
```
Comment (andrewpai) [inline astro/src/content/docs/get-started/download-and-install/setup-wizard.mdx:2]: It's been deemed that `First-time Setup` is the grammatically correct way of writing this. We could either leave this for now since it matches what's in the app and screen shots, or we could update it in the headings now.

File: astro/src/content/docs/get-started/run-in-the-cloud/setup-wizard.mdx [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Setup Wizard & First Login
+title: Setup Wizard & First Time Setup
```
Comment (andrewpai) [inline astro/src/content/docs/get-started/run-in-the-cloud/setup-wizard.mdx:2]: It's been deemed that First-time Setup is the grammatically correct way of writing this. We could either leave this for now since it matches what's in the app and screen shots, or we could update it in the headings now.

File: astro/src/content/docs/_shared/_setup-wizard.mdx [hunk 4]
```diff
@@ -14,34 +14,59 @@ Once the setup wizard is complete you will be logged into FusionAuth using the c
...
+
 To complete the setup wizard you must toggle the Accept button after reading the FusionAuth license agreement. You won't find any surprise strings attached here, just the normal lawyer speak for CYA.
 
-#### Stay informed
 Optionally stay informed by opting-in to receive FusionAuth technical announcements, tutorials and other technical focused emails. You may opt-out at any time in the future, or opt-in at a later date by visiting [fusionauth.io](/) and finding the "Subscribe for updates" form at the bottom of the page.
 
-## Complete Setup
+Your feedback on how you learned about us helps us better serve our customers.
+
+## Complete First Time Setup
```
Comment (andrewpai) [inline astro/src/content/docs/_shared/_setup-wizard.mdx:29]: It's been deemed that First-time Setup is the grammatically correct way of writing this. We could either leave this for now since it matches what's in the app and screen shots, or we could update it in the headings now.

File: astro/src/content/docs/_shared/_setup-wizard.mdx [hunk 5]
```diff
@@ -14,34 +14,59 @@ Once the setup wizard is complete you will be logged into FusionAuth using the c
...
 
-Once the Setup Wizard has logged you into FusionAuth you may begin using FusionAuth. On the dashboard you will see the remaining setup items.
-This includes setting up an API key, creating an Application and completing the SMTP Email configuration.
+Once the Setup Wizard has logged you into FusionAuth you may begin using FusionAuth. On the dashboard you will see a panel for new users.
```
Comment (andrewpai) [inline astro/src/content/docs/_shared/_setup-wizard.mdx:31]:
```suggestion
Once the Setup Wizard has logged you into FusionAuth you may begin using the application. On the dashboard you will see a panel for new users.
```

File: astro/src/content/docs/_shared/_setup-wizard.mdx [hunk 6]
```diff
@@ -14,34 +14,59 @@ Once the setup wizard is complete you will be logged into FusionAuth using the c
...
+Clicking Setup will guide you through creating an Application, adding an API key, configuring the email server, and optionally activating your license.
```
Comment (andrewpai) [inline astro/src/content/docs/_shared/_setup-wizard.mdx:32]:
```suggestion
Clicking `Setup` will guide you through creating an Application, adding an API key, configuring the email server, and optionally activating your license.
```

File: astro/src/content/docs/_shared/_setup-wizard.mdx [hunk 7]
```diff
@@ -14,34 +14,59 @@ Once the setup wizard is complete you will be logged into FusionAuth using the c
...
 
-<img src="/img/docs/get-started/download-and-install/complete-setup.png" alt="Complete Setup" width="1200" role="shadowed" />
+<img src="/img/docs/get-started/download-and-install/first-time-setup-panel.png" alt="First Time Setup Panel" width="1200" role="bottom-cropped" />
```
Comment (andrewpai) [inline astro/src/content/docs/_shared/_setup-wizard.mdx:34]: It's been deemed that First-time Setup is the grammatically correct way of writing this. We could either leave this for now since it matches what's in the app and screen shots, or we could update it in the headings now.

File: astro/src/content/docs/_shared/_setup-wizard.mdx [hunk 8]
```diff
@@ -14,34 +14,59 @@ Once the setup wizard is complete you will be logged into FusionAuth using the c
...
 
-### Missing Application
+### Create an application
 FusionAuth can be configured with one to many Applications. A FusionAuth Application represents a secured resource, it should be thoughtfully
-named but the name may be modified at a later time. See the [Application overview](/docs/get-started/core-concepts/applications) for additional details.
+named but the name may be modified at a later time.
+
+The Authorized redirect URL should match where your users will be redirected after logging in with FusionAuth.
```
Comment (andrewpai) [inline astro/src/content/docs/_shared/_setup-wizard.mdx:40]: Should we mention that this is where the OAuth token exchange will happen, too? That's pretty technical for here and some people may not know what that means yet, but for those that do it could be helpful.

**Review by andrewpai — APPROVED** [review]

File: astro/src/content/blog/single-sign-on-django-fusionauth.mdx (hunk 1 above)
Comment (bhalsey) [reply astro/src/content/blog/single-sign-on-django-fusionauth.mdx]: fixed.

File: astro/src/content/docs/_shared/_setup-wizard.mdx (hunk 8 above)
Comment (bhalsey) [reply astro/src/content/docs/_shared/_setup-wizard.mdx:40]: That feels out of place here.  My guess is the intended audience won't know that.  But happy to revisit.

**Comment by bhalsey** [comment]
I'll hold off on switching to the grammatically correct First-time Setup until we update the app so the screenshots and docs are consistent.
